distributed/utils_test.py

```diff
@@ -1770,7 +1770,12 @@ def clean(threads=not WINDOWS, instances=True, timeout=1, processes=True):
                     with check_active_rpc(loop, timeout):
                         reset_config()
 
-                        dask.config.set({"distributed.comm.timeouts.connect": "5s"})
+                        dask.config.set(
+                            {
+                                "distributed.comm.timeouts.connect": "5s",
+                                "distributed.admin.tick.interval": "500 ms",
+                            }
+                        )
```

Comment:
This config setting is not side effect free, i.e. it doesn't reset itself properly afterwards. That causes tests to run with different configurations depending on the order of test execution.
I tried to fix it earlier but that causes **many** other failures. I'd prefer not adding more config params to this

Reply:
I was curious why we weren't using the context manager here.  I'd be tempted to use it and see what happens.  I'm going to push up a commit now to see what breaks.  Probably I'm walking down a bad path, but we'll see what happens :)

Reply:
Tests seem happy here.  Do you still have concerns?  I wouldn't be surprised to learn that there was untested conflicts somewhere, but I'm hoping not :)

Reply:
if the tests are happy, I'm happy. this is how it is supposed to be after all